app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -246,7 +246,7 @@ private fun DeleteAccountDialog(
 		onDismissRequest = onDismiss,
 		confirmButton = {
 			TextButton(onClick = onConfirm) {
-				Text(stringResource(R.string.ConfirmButton).uppercase())
+				Text(stringResource(R.string.DeleteButton).uppercase())
```

Comment:
Could you link the source with information what guideline says about our case?

Reply:
Headline: https://m3.material.io/components/dialogs/guidelines#0cff8b6b-d4f3-4442-8194-31212f5d1a12
Button: https://m3.material.io/components/dialogs/guidelines#c395b8a7-a125-42f4-9d19-77f535b2c873 
Maybe it's not a strict guideline about button text but looks like good UX practice in multiple examples in official documentation.

Reply:
Nice, thanks :)